### DISCUSSION TITLE
Voluntary software translation

### DISCUSSION THREAD

**Discussion by Danik2343** [86Box/86Box#1876] [General]
Please add the ability to translate 86Box into other languages ​​using special services, I personally can help with Russian, I am a native speaker of Russian.

**Comment by dhrdlicka** [comment]
Latest builds of 86Box already include translations to other languages, unless you are asking for that feature to be changed in some way.

**Reply by OBattler** [reply]
Yes, but I haven't heard from that person since, so if someone else can do it, I don't see why not.
